backend/testing/configs/config.yaml

```diff
@@ -96,3 +100,27 @@ rules:
       '*'
   mitigation:
       manual
+- prefixes:
+  - *test_comms_24
+  origin_asns:
+  - *comm_test_origin
+  neighbors:
+      '*'
+  community_annotations:
+  - critical:
```

Comment:
what happens if you put same community on all levels? which one is used?

Reply:
the most critical, according to the order it has been put in the list.